conn/raft_server.go

```diff
@@ -120,20 +121,22 @@ func (p *proposals) Done(key string, err error) {
 	pd.Ch <- err
 }
 
-func (w *RaftServer) GetNode() *Node {
+func (w *RaftServer) getNode() *Node {
```

Comment:
Rename to `Node()`

Reply:
Renamed to node since there's no need to export this method.